### PR TITLE
SITL: added StratoBlimp simulator

### PR DESCRIPTION
**Pull request by tridge**
this simulates a stratospheric` blimp, with 2 tilting motors
based on discussions with https://www.stratoship.au/

this also adds the ability to add parameter tables to our backend physics models, which makes changing the parameters for a model a lot easier

### CODE REVIEW

File: Tools/autotest/default_params/stratoblimp.parm [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+SERVO3_FUNCTION 73
+SERVO3_MIN 1000
+SERVO3_MAX 2000
+
+SERVO4_FUNCTION 74
+SERVO4_MIN 1000
+SERVO4_MAX 2000
+
+# no need for
```
Comment (MichelleRos) [inline Tools/autotest/default_params/stratoblimp.parm:13]: Maybe complete the sentence?

**Review by MichelleRos — APPROVED** [review]
LGTM other than the comment.

File: Tools/autotest/default_params/stratoblimp.parm (hunk 1 above)
Comment (tridge) [reply Tools/autotest/default_params/stratoblimp.parm]: fixed

File: libraries/SITL/SIM_StratoBlimp.cpp [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+    // @Description: thrust at max throttle for one motor
+    // @Units: N
+    AP_GROUPINFO("MOT_THST", 4, StratoBlimp,  motor_thrust, 145),
+
+    // @Param: DRAG_FWD
+    // @DisplayName: drag in forward direction
+    // @Description: drag on X axis
+    AP_GROUPINFO("DRAG_FWD", 5, StratoBlimp,  drag_fwd, 0.27),
+
+    // @Param: DRAG_FWD
```
Comment (Hwurzburg) [inline libraries/SITL/SIM_StratoBlimp.cpp:63]: this is a duplicate param name! causing wiki build errors in parameters docs...please fix! should be DRAG_UP I think

**Review by Hwurzburg — COMMENTED** [review]
ERROR in naming!

File: libraries/SITL/SIM_StratoBlimp.cpp (hunk 2 above)
Comment (tridge) [reply libraries/SITL/SIM_StratoBlimp.cpp:63]: fixed, thanks

**Pull request merged**